merchantapi/src/main/java/fi/hel/verkkokauppa/configuration/mapper/PaytrailMerchantMappingMapper.java

```diff
@@ -0,0 +1,17 @@
+package fi.hel.verkkokauppa.configuration.mapper;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import fi.hel.verkkokauppa.configuration.api.merchant.dto.PaytrailMerchantMappingDto;
+import fi.hel.verkkokauppa.configuration.model.merchant.PaytrailMerchantMapping;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PaytrailMerchantMappingMapper {
```

Comment:
We have abstract class to do converting, could you use that in here? it comes from common AbstractModelMapper -> Example usage https://github.com/City-of-Helsinki/verkkokauppa-core/blob/074e5ccba124b875c2dd321ee47ec4d893eba0af/merchantapi/src/main/java/fi/hel/verkkokauppa/configuration/mapper/ConfigurationMapper.java#L11

Reply:
That class can handle list of objects etc

Reply:
Check this again?